src/parquet/types.h

```diff
@@ -276,6 +276,8 @@ std::string logical_type_to_string(LogicalType::type t);
 std::string type_to_string(Type::type t);
 
 std::string FormatStatValue(Type::type parquet_type, const char* val);
+
+size_t get_value_byte_size(Type::type t);
```

Comment:
Think we need to fix the naming for the other methods in this file to be in line with Google SG (e.g. `TypeToString` and `GetTypeByteSize`)